content/chainguard/libraries/overview.md

```diff
@@ -59,8 +59,12 @@ contents into our production environment.
 ## Introduction 
 
 Chainguard Libraries builds all available libraries from source code in the
-secure Chainguard infrastructure and makes them available for you. It removes
-all the software supply chain problems for libraries:
+Chainguard Factory and makes them available for you. Chainguard Factory is a
```

Comment:
I would add some language like below about how it's Chainguard's internal build infra

```suggestion
Chainguard Factory and makes them available for you. Chainguard Factory represents Chainguard's internal tooling that enable a
```

Reply:
Done